test/karma-system-config.js

```diff
@@ -116,7 +116,7 @@ System.config({
     '@angular/material/dialog/testing/shared.spec': 'dist/packages/material/dialog/testing/shared.spec.js',
     '@angular/material/divider': 'dist/packages/material/divider/index.js',
     '@angular/material/expansion': 'dist/packages/material/expansion/index.js',
-    '@angular/material/form-field': 'dist/packages/material/form-field/index.js',
+    '@angular/material/form-field':  'dist/packages/material/form-field/index.js',
```

Comment:
need to add `/testing` and `/testing/shared.spec`

Reply:
done.